src/clr-angular/emphasis/alert/alert.spec.ts

```diff
@@ -79,25 +79,25 @@ export default function(): void {
     });
 
     it('extends the alert type classes when clrAlertType is set', () => {
-      // default alert-info class
+      // default info class
       expect(compiled.querySelector('.alert-info')).not.toBeNull();
 
-      // set alert-danger
-      fixture.componentInstance.type = 'alert-danger';
+      // set danger
+      fixture.componentInstance.type = 'danger';
       fixture.detectChanges();
 
       expect(compiled.querySelector('.alert-info')).toBeNull();
       expect(compiled.querySelector('.alert-danger')).not.toBeNull();
 
-      // set alert-warning
-      fixture.componentInstance.type = 'alert-warning';
+      // set warning
+      fixture.componentInstance.type = 'warning';
       fixture.detectChanges();
 
       expect(compiled.querySelector('.alert-danger')).toBeNull();
       expect(compiled.querySelector('.alert-warning')).not.toBeNull();
 
-      // set alert-success
-      fixture.componentInstance.type = 'alert-success';
```

Comment:
Straight from our docs:
<img width="873" alt="screen shot 2018-10-18 at 12 38 35" src="https://user-images.githubusercontent.com/4647197/47169929-c5e6c300-d2d2-11e8-81f1-7cbb964a70f6.png">

It uses every single deprecated syntax. I'm okay still cutting all this for 1.0, but we need to at least update the docs before we can actually cut them.

Reply:
Yes we need to fix docs, but thats an oversight when it was deprecated to not update the docs accordingly.

Reply:
Not sure it was an oversight or not. We had no timeline for the deprecation and back then had no strategy for handling deprecations like this.